test/qr_code.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
```

Comment:
**issue**: File name must end with `_test` to be included in `flutter test` run.